drivers/bridge/bridge.go

```diff
@@ -72,6 +71,9 @@ type containerConfiguration struct {
 type bridgeEndpoint struct {
 	id              types.UUID
 	intf            *sandbox.Interface
```

Comment:
As I understand, sandbox.Interface is changed from a struct to an interface and hence all the fields that were conveniently carried from sandbox.Interface is not directly defined under bridgeEndpoint. In that case, is there a reason why the intf member is not removed ?